dynawo/sources/API/CRV/CRVCurvesCollection.h

```diff
@@ -248,6 +248,12 @@ class CurvesCollection {
    */
   iterator end();
 
+  /**
+   * @brief
+   * @return
+   */
+  iterator find(const std::string &model, const std::string &variable);
```

Comment:
Do you need to update the value ? If not, use the const_iterator version

Reply:
Method has been removed. A local map is created in `Simulation` to merge CRV and FSV definitions.